doc/topics/tutorials/minionfs.rst

```diff
@@ -4,10 +4,27 @@
 MinionFS Backend Walkthrough
 ============================
 
+Propagating Files
+============================
```

Comment:
We can cut down on the number of `=` signs, they just need to be the same length as the line they are below so the character count is the same.